relay-server/src/metrics_extraction/transactions.rs

```diff
@@ -115,7 +128,7 @@ pub fn extract_transaction_metrics(
                 };
 
                 push_metric(Metric {
-                    name: format!("breakdown.{}.{}", breakdown, name),
+                    name: metric_name(&["breakdowns", breakdown, name]),
```

Comment:
Note that this renames metrics from `breakdown.foo` to `breakdowns.foo`, to be consistent with `measurements`.